crates/rpc/src/rpc.rs

```diff
@@ -76,13 +76,15 @@ impl Rpc for Server {
                      description,
                      favicon_url,
                      logo_url,
+                     created_at,
                  }| RpcStorefront {
                     owner_address: owner_address.into_owned(),
                     subdomain: subdomain.into_owned(),
                     title: title.into_owned(),
                     description: description.into_owned(),
                     favicon_url: favicon_url.into_owned(),
                     logo_url: logo_url.into_owned(),
+                    created_at,
```

Comment:
```suggestion
                    created_at: created_at.map(Timestamp::from_utc),
```

In accordance with the suggestion to `rpc_models.rs`